test cases/rust/26 cargo lock suggest dependency/test.json

```diff
@@ -0,0 +1,7 @@
+{
+  "stdout": [
+    {
+      "line": "test cases/rust/26 cargo lock suggest dependency/meson.build:3:0: ERROR: Rust dependency \"bar-1.0-rs\" not found, but following versions were found using Cargo.lock import: \"bar-1-rs\", \"bar-2-rs\", \"bar-0-rs\", \"bar-0.8-rs\""
```

Comment:
I'm not sure whether this really needs a test. All we are testing is the warning message, no? Does that really need an integration test?

Reply:
If someone changes the cargo.lock import feature later, it would be useful to notice this. But if you don't see any value in this test, I can remove it too.